src/server/schema/stripe.schema.ts

```diff
@@ -73,7 +73,13 @@ export type ProductMetadata = z.infer<typeof productMetadataSchema>;
 export const productMetadataSchema = z
   .object({
     vaultSizeKb: z.coerce.number().positive().optional(),
-    buzz: z.coerce.number().positive().optional(),
+    badge: z.string().optional(),
+    monthlyBuzz: z.coerce.number().positive().optional(),
+    animatedBadge: z
+      .string()
+      .transform((val) => val === 'true')
```

Comment:
We've got a `booleanString` helper method for this, see if that helps
```suggestion
    animatedBadge: booleanString()
```